### ISSUE TITLE
154

### ISSUE THREAD

**Issue by eclee25** [HopkinsIDD/cholera-database#68]

**Comment by QLLZ** [comment]
Hi @mooresea, I didn't find the start date for the 2012 outbreak from the source document, where did you get the start date for the outbreak?

**Comment by mooresea** [comment]
@QLLZ  Do you have the attached source document? The bottom chart has weekly cases, and I believe that I estimated the start date from the epi week with the first case/death.
[Actualização da situação concernente à epidemia de Cólera na Guiné 04 jan 13.pdf](https://github.com/HopkinsIDD/cholera-database/files/3601663/Actualizacao.da.situacao.concernente.a.epidemia.de.Colera.na.Guine.04.jan.13.pdf)

**Comment by QLLZ** [comment]
@mooresea yeah, I think you are right. I saw the table and graph but thought they were unrelated. However, I just noticed the total number of deaths were the same and the total number of cases were similar between the table and the graph. They should be corresponding to each other. Thank you!
